### ISSUE TITLE
5.2.0 build fail

### ISSUE THREAD

**Issue by jubalh** [mozilla/rr#2391]
When I try to compile 5.2.0 on openSUSE Tumbleweed I get:

```
[   23s] [ 68%] Building CXX object CMakeFiles/rr.dir/src/kernel_abi.cc.o                                                                                     
[   23s] /usr/bin/c++  -DRR_VERSION=\"5.2.0\" -Drr_EXPORTS -I/home/abuild/rpmbuild/BUILD/rr-5.2.0/include -I/home/abuild/rpmbuild/BUILD/rr-5.2.0/third-party/p
roc-service -I/home/abuild/rpmbuild/BUILD/rr-5.2.0/third-party/brotli/include -I/home/abuild/rpmbuild/BUILD/rr-5.2.0/build  -O2 -Wall -D_FORTIFY_SOURCE=2 -fst
ack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -g -DNDEBUG -msse2 -D__MMX__ -D__SSE__ -D__SSE2
__ -D__USE_LARGEFILE64 -pthread -g3 -D__STDC_LIMIT_MACROS -D__STDC_FORMAT_MACROS -std=c++11  -pthread -O2 -g -DNDEBUG   -Wall -Wextra -O2 -UDEBUG -DNDEBUG -o 
CMakeFiles/rr.dir/src/kernel_abi.cc.o -c /home/abuild/rpmbuild/BUILD/rr-5.2.0/src/kernel_abi.cc                                                               
[   23s] /home/abuild/rpmbuild/BUILD/rr-5.2.0/src/kernel_abi.cc:54:58: error: static assertion failed: type ::ucontext_t not correctly defined                
[   23s]    54 |   static_assert(Verifier<arch_, system_type_, rr_type_>::same_size,            \                                                             
[   23s]       |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~                                                                           
[   23s] /home/abuild/rpmbuild/BUILD/rr-5.2.0/src/kernel_abi.h:1813:3: note: in expansion of macro 'RR_VERIFY_TYPE_ARCH'                                      
[   23s]  1813 |   RR_VERIFY_TYPE_ARCH(SupportedArch::x86_64, ::ucontext_t, ucontext_t);                                                                      
[   23s]       |   ^~~~~~~~~~~~~~~~~~~     
```

**Comment by rocallahan** [comment]
Fixed on master by 53c5bd72bae089616a3ca626b8af240481d70e6f

**Comment by jubalh** [comment]
Thanks!

with https://github.com/mozilla/rr/commit/53c5bd72bae089616a3ca626b8af240481d70e6f and https://github.com/mozilla/rr/commit/ef33fa1054734752975e0253c55e3200d0c22a63 the 5.2.0 release is compilable under openSUSE Tumbleweed.
